public/locales/en.json

```diff
@@ -102,7 +102,7 @@
   "investorClaimAmount": "Claimable: {{claimable}}",
   "investorRemainingAmount": "Total SDL: {{remaining}}",
   "language": "Language",
-  "learnMore": "Learn more",
+  "learnMore": "<Learn more>",
```

Comment:
We would probably prefer to keep formatting out of translations

Reply:
Reverted